.eslintrc

```diff
@@ -1,211 +1,6 @@
-    "no-inner-declarations": [2, "functions"],
-    "no-invalid-regexp": 2,
-    "no-invalid-this": 0,
-    "no-irregular-whitespace": 2,
-    "no-iterator": 2,
-    "no-label-var": 2,
-    "no-labels": 0,
-    "no-lone-blocks": 2,
-    "no-lonely-if": 2,
-    "no-loop-func": 0,
-    "no-mixed-requires": [2, true],
-    "no-mixed-spaces-and-tabs": 2,
-    "no-multi-spaces": 2,
-    "no-multi-str": 2,
-    "no-multiple-empty-lines": 0,
-    "no-native-reassign": 0,
-    "no-negated-in-lhs": 2,
-    "no-nested-ternary": 0,
-    "no-new": 2,
-    "no-new-func": 0,
-    "no-new-object": 2,
-    "no-new-require": 2,
-    "no-new-wrappers": 2,
-    "no-obj-calls": 2,
-    "no-octal": 2,
-    "no-octal-escape": 2,
-    "no-param-reassign": 2,
-    "no-path-concat": 2,
-    "no-plusplus": 0,
-    "no-process-env": 0,
-    "no-process-exit": 0,
-    "no-proto": 2,
-    "no-redeclare": 2,
-    "no-regex-spaces": 2,
-    "no-restricted-modules": 0,
-    "no-return-assign": 2,
-    "no-script-url": 2,
-    "no-self-compare": 0,
-    "no-sequences": 2,
-    "no-shadow": 2,
-    "no-shadow-restricted-names": 2,
-    "no-spaced-func": 2,
-    "no-sparse-arrays": 2,
-    "no-sync": 2,
-    "no-ternary": 0,
-    "no-this-before-super": 2,
-    "no-throw-literal": 2,
-    "no-trailing-spaces": 2,
-    "no-undef": 2,
-    "no-undef-init": 2,
-    "no-undefined": 0,
-    "no-underscore-dangle": 0,
-    "no-unexpected-multiline": 2,
-    "no-unneeded-ternary": 2,
-    "no-unreachable": 2,
-    "no-unused-expressions": 2,
-    "no-unused-vars": [2, {"vars": "all", "args": "after-used"}],
-    "no-use-before-define": 0,
-    "no-useless-call": 2,
-    "no-var": 0,
-    "no-void": 2,
-    "no-warning-comments": 0,
-    "no-with": 2,
-    "object-curly-spacing": [0, "always"],
-    "object-shorthand": [2, "always"],
-    "one-var": [2, "never"],
-    "operator-assignment": [2, "always"],
-    "operator-linebreak": [2, "after"],
-    "padded-blocks": 0,
-    "prefer-const": 0,
-    "prefer-reflect": 0,
-    "prefer-spread": 0,
-    "quote-props": [2, "as-needed"],
-    "quotes": [2, "single"],
-    "radix": 2,
-    "require-yield": 2,
-    "semi": [2, "always"],
-    "semi-spacing": [2, {"before": false, "after": true}],
-    "sort-vars": 0,
-    "space-after-keywords": [2, "always"],
-    "space-before-blocks": [2, "always"],
-    "space-before-function-paren": [2, {"anonymous": "always", "named": "never"}],
-    "space-in-parens": 0,
-    "space-infix-ops": [2, {"int32Hint": false}],
-    "space-return-throw-case": 2,
-    "space-unary-ops": [2, {"words": true, "nonwords": false}],
-    "spaced-comment": [2, "always"],
-    "strict": 0,
-    "use-isnan": 2,
-    "valid-jsdoc": 0,
-    "valid-typeof": 2,
-    "vars-on-top": 0,
-    "wrap-iife": 2,
-    "wrap-regex": 0,
-    "yoda": [2, "never", {"exceptRange": true}],
-    "react/jsx-uses-react": 1
+  "extends":"airbnb",
+  "rules":{
+    "react/jsx-no-bind":[2, {"allowArrowFunctions":true}]
```

Comment:
Also make this to `react/jsx-no-bind:0`. It doesn't add much cost. JS VMs are pretty good at optimizing these kind of stuff.